recipes/dagmc/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{% set name = "dagmc" %}
+{% set version = "3.2" %}
+{% set sha256 = "becd61148aa4f9a3a312eb69e33d01779797007cc13715d60bb650ad5e71b183" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/svalinn/DAGMC/archive/v{{ version }}-rc.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: True  # [win]
+  detect_binary_files_with_prefix: true
+
+requirements:
+  build:
+    - cmake
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - {{ compiler("fortran") }}
+  host:
+    - python
+    - hdf5
+    - moab 5.*
+  run:
+    - python
+    - hdf5
+    - moab 5.*
+    - libopenblas
+    - zlib
+
+test:
+   commands:
+     - make_watertight --help
+
+about:
+  home: https://svalinn.github.io/DAGMC/
+  license: BSD
```

Comment:
Which kind of BSD license? Please use the SPDX classifier here and add `license_family: BSD`